visual-tests/src/test-fixtures/series/bar.js

```diff
@@ -25,18 +25,32 @@
     });
 
     // Create scale for y axis
-    var priceScale = d3.scale.linear()
+    var priceScale1 = d3.scale.linear()
         .domain(fc.utilities.extent(data, ['close']))
-        .range([height, 0])
-        .nice();
+        .range([0, height / 2]);
 
-    var bar = fc.series.bar()
+    var bar1 = fc.series.bar()
+        .y0Value((extent[1] - extent[0]) / 3)
         .xScale(dateScale)
-        .yScale(priceScale);
+        .yScale(priceScale1);
 
     // Add it to the chart
     container.append('g')
         .datum(data)
-        .call(bar);
+        .call(bar1);
+
+    // Create scale for y axis
```

Comment:
If you haven't already seen, D3's scales have [`.copy()` ](https://github.com/mbostock/d3/wiki/Quantitative-Scales#linear_copy), which:

> Returns an exact copy of this linear scale. Changes to this scale will not affect the returned scale, and vice versa.

So you could do something like this (which looks quite neat to me):

```js
var priceScale2 = priceScale1.copy()
    .range([height / 2, height]);
```